Makefile

```diff
@@ -523,4 +523,6 @@ arch:
 e2e_test: tools
 	@echo "running e2e tests"
 	cd ./test/e2e && \
-	ginkgo -r $(E2E_TEST) -v -keep-going -- -yk-namespace "yunikorn" -kube-config $(KUBECONFIG)
```

Comment:
I guess those changes are used to run test locally. Please revert it before merging :)